plugins/woocommerce-admin/client/marketplace/utils/functions.tsx

```diff
@@ -314,6 +317,31 @@ function installProduct( subscription: Subscription ): Promise< void > {
 	} );
 }
 
+function updateProduct( subscription: Subscription ): Promise< void > {
+	return wpAjax( 'update-' + subscription.product_type, {
+		slug: subscription.local.slug,
+		plugin: subscription.local.path,
+		theme: subscription.local.path,
```

Comment:
It seems odd to pass both these parameters – the product is either a plugin or a theme – but I suppose you have a good reason for it.

Reply:
Just for simplicity of the implementation. 
I will update it to send just the param it needs.